debian/vyos-1x.install

```diff
@@ -6,7 +6,7 @@ etc/netplug
 etc/opennhrp
 etc/modprobe.d
 etc/ppp
-etc/rsyslog.d
```

Comment:
I'd like to leave it for custom logs.

Reply:
So this change does not create the directory nor remove it. It only tells the debian installer that if the directory exists during build time, it will be shipped in the corresponding deb package. No more, no less.